NOTES.ANDROID

```diff
@@ -0,0 +1,67 @@
+
+ NOTES FOR THE ANDROID PLATFORMS
+ ===============================
+
+ Requirement details
+ -------------------
+
+ Besides basic tools like perl and make you'll need to download Android
+ NDK. It's avialable for Linux, MacOS X and Windows, but only Linux
+ version was actually tested. There is no reason to believe that MacOS X
+ wouldn't work. And as for Windows it's unclear which "shell" would work
+ [best], MSYS2 might have best chances. NDK version should play lesser
+ role, the goal is to support any recent enough.
+
+ Configuration
+ -------------
+
+ Android is naturally cross-compiled target and you can't use ./config.
+ You have to use ./Configure and name your target explicitly, there are
+ android-arm, android-arm64, android-mips, android-mip64, android-x86
+ and android-x86_64. You might expect you would be required to pass
+ --cross-compile-prefix or set $CROSS_COMPILE environment variable, but
+ not this time, it will be "calculated" automatically based on chosen
+ platform. Though you'd still need to know it, the prefix, for $PATH
+ adjustment, that would allow to invoke $(CROSS_COMPILE)gcc and company.
+ [Yes, it can be summarized as "a bit messy."] Besides PATH adjustment
+ you need to set $ANDROID_NDK environment that points at NDK directory
+ as /some/where/android-ndk-<ver>.
+
+ Previous instructions required you to set even CROSS_SYSROOT variable
+ [to $ANDROID_NDK/platforms/android-<api>/arch-<arch>]. It's still
+ recognized [and even required for some legacy targets], but if not set,
+ it's "caclulated" and set to the latest Android platform available with
+ appointed NDK automatically. In such case if you need to target older
```

Comment:
Remove 'In such case', just let the sentense start with 'If you need to targer older ...'